index.html

```diff
@@ -0,0 +1,724 @@
+    </nav>
+
+    <nav id="nav_bar3">
+        <div>
+            <div id="nav_bar_logo">
+                <h2>TNW</h2>
+            </div>
+            <div class="button_search"></div>
+            <div id="cont_search_menu">
+                <i id="search_int" class="fas fa-search"></i>
+                <i id="menu_color" class="fas fa-bars"></i>
+            </div>
+        </div>
+    </nav>
+    <nav class="posting-nav">
+        <div class="posting">
+            Calling young digital talent in the Netherlands – Be recognized in the T500 →
+        </div>
+    </nav>
+
+    <section id="section1">
+        <div id="section1_box1">
+            <div class="big-logo">
+                <img alt="" src="./images/big-logo.svg">
+                <div class="date">Tuesday — January 26, 2021</div>
+            </div>
+            <h1><a id="image_title" href="#">Opal cuts off your apps' internet access to help you stop doomscrolling</a>
+            </h1>
+            <div class="flex_text">
+                <h4>by</h4>
+                <a href="#">Ivan Mehta</a>
+                <i class="far fa-clock"></i>
+                <p>1 hour ago</p>
+                <i class="fas fa-share-alt"></i>
+                <p>6</p>
+            </div>
+        </div>
+        <div id="section1_box2">
+            <h1><a class="image_title2" href="#">Google unveils privacy-friendly AI cookie killer</a></h1>
+            <div class="flex_text">
+                <p>by</p>
+                <a href="#">Thomas Macaulay</a>
+                <i class="far fa-clock"></i>
+                <p>2 hours ago</p>
+                <i class="fas fa-share-alt"></i>
+                <p>4</p>
+            </div>
+        </div>
+        <div id="section1_box3">
+            <h1><a class="image_title2" href="#">This Mac app automatically detects your duplicate files</a></h1>
+            <div class="flex_text">
+                <p>by</p>
+                <a href="#">Ivan Mehta</a>
+                <i class="far fa-clock"></i>
+                <p>2 hours ago</p>
+                <i class="fas fa-share-alt"></i>
+                <p>5</p>
+            </div>
+        </div>
+    </section>
+
+    <div id="title1">
+        <h3><a id="title_section1" href="#"><strong>Latest news ></strong></a></h3>
+    </div>
+    <section id="section2">
+        <h2 class="none">title</h2>
+        <div class="section2_items">
+            <img alt="image"
+                src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/california-car-ev-hydrogen-future-electruc-fcev-ev-480x270.png">
+            <div>
+                <a class="section2_links" href="">Want to drive a hydrogen EV in the US? Move to California</a>
+                <div>
+                    <a class="special_text" href="#">Matthew Beedham</a>
+                    <p class="special_text">· 22 minutes ago</p>
+                </div>
+            </div>

```

Comment:
Each article in the Latest News section should have an image and a title (h1--h6) tag, both of them with a link (a tag). i.e.

```
<h3>
  <a href="#" class="img-text">
    <p> Want to drive a hydrogen EV in the US? Move to California</p>
  </a>
</h3>
```

Reply:
Thanks for the suggestions. These changes have been made